.github/CODEOWNERS

```
@@ -0,0 +1,17 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, users and
+# teams which are put in owners will be requested for review
+# when someone opens a pull request.
+
+# This is base configuration. These owners could review the
+# changes in all files in this repository.
+* @cognifloyd
```

```suggestion
* @StackStorm-Exchange/tsc @cognifloyd
```

I guess there are more codeowners here.  TSC group will be also automatically requested for new PR reviews.

---

I cannot be responsible for adding the TSC to something where they will be pinged for every change. Without fail, whenever I start debugging something on the exchange or start working on a mass update, someone on the TSC complains about the notifications.